### DISCUSSION TITLE
More Sdxl Resolutions Request

### DISCUSSION THREAD

**Discussion by boricuapab** [runew0lf/RuinedFooocus#42] [Ideas]
Would be awesome to have more of the sdxl resolutions options similar to the sdxl latent resolution node in comfy.

![sdxlEmptyLatentResolutions](https://github.com/runew0lf/RuinedFooocus/assets/61924816/12dc4c6f-0334-4818-b0ff-2955f96f1dfe)

The smaller 512x512 would be great to have as options to just iterate through prompts faster for using with some of the community sdxl models that support resolutions lower than 1024x1024.

**Comment by runew0lf** [comment]
I love the minimalistic list of the current setup we have at the moment. however under the `settings` folder you can see a `resolutions.json` if you want to add more resolutions you can add them into there, it wont get overwritten by update and you can chop/change customize as much as you like¬
